modules/custom-css/custom-css/js/css-editor.js

```diff
@@ -31,53 +31,60 @@
 	addLoadEvent( safecssInit );
 } )( jQuery );
 
-jQuery( function ( $ ) {
-	$( '.edit-preprocessor' ).bind( 'click', function ( e ) {
+jQuery( function( $ ) {
+	$( '.edit-preprocessor' ).bind( 'click', function( e ) {
 		e.preventDefault();
 
 		$( '#preprocessor-select' ).slideDown();
 		$( this ).hide();
 	} );
 
-	$( '.cancel-preprocessor' ).bind( 'click', function ( e ) {
+	$( '.cancel-preprocessor' ).bind( 'click', function( e ) {
 		e.preventDefault();
 
-		$( '#preprocessor-select' ).slideUp( function () {
+		$( '#preprocessor-select' ).slideUp( function() {
 			$( '.edit-preprocessor' ).show();
 			$( '#preprocessor_choices' ).val( $( '#custom_css_preprocessor' ).val() );
 		} );
 	} );
 
-	$( '.save-preprocessor' ).bind( 'click', function ( e ) {
+	$( '.save-preprocessor' ).bind( 'click', function( e ) {
 		e.preventDefault();
 
 		$( '#preprocessor-select' ).slideUp();
 		$( '#preprocessor-display' ).text( $( '#preprocessor_choices option:selected' ).text() );
-		$( '#custom_css_preprocessor' ).val( $( '#preprocessor_choices' ).val() ).change();
+		$( '#custom_css_preprocessor' )
+			.val( $( '#preprocessor_choices' ).val() )
+			.change();
 		$( '.edit-preprocessor' ).show();
 	} );
 
-	$( '.edit-css-mode' ).bind( 'click', function ( e ) {
+	$( '.edit-css-mode' ).bind( 'click', function( e ) {
 		e.preventDefault();
 
 		$( '#css-mode-select' ).slideDown();
 		$( this ).hide();
 	} );
 
-	$( '.cancel-css-mode' ).bind( 'click', function ( e ) {
+	$( '.cancel-css-mode' ).bind( 'click', function( e ) {
 		e.preventDefault();
 
-		$( '#css-mode-select' ).slideUp( function () {
+		$( '#css-mode-select' ).slideUp( function() {
 			$( '.edit-css-mode' ).show();
-			$( 'input[name=add_to_existing_display][value=' + $( '#add_to_existing' ).val() + ']' ).attr( 'checked', true );
+			$( 'input[name=add_to_existing_display][value=' + $( '#add_to_existing' ).val() + ']' ).attr(
+				'checked',
+				true
+			);
 		} );
 	} );
 
-	$( '.save-css-mode' ).bind( 'click', function ( e ) {
+	$( '.save-css-mode' ).bind( 'click', function( e ) {
 		e.preventDefault();
 
 		$( '#css-mode-select' ).slideUp();
-		$( '#css-mode-display' ).text( $( 'input[name=add_to_existing_display]:checked' ).val() == 'true' ? 'Add-on' : 'Replacement' ); // jshint ignore:line
+		$( '#css-mode-display' ).text(
+			$( 'input[name=add_to_existing_display]:checked' ).val() === 'true' ? 'Add-on' : 'Replacement'
```

Comment:
Another example of `==` being replaced with `===`

Reply:
This is not prettier, this was a manual change I did to fix the lint.

Broken out into #11576, I'll remove from this PR